### PR TITLE
Remove videoPlayback feature flag

### PR DESCRIPTION
**Pull request by bvaughn**
This feature flag interferes with plans to enable users to play the video outside of the focus region (which may also include unloaded regions). It was originally added in #2471 and disabled by default in #3110.

Without this feature flag, the `VideoPlayer` component is no longer used, so this PR removes it as well.

### CODE REVIEW

**Review by jcmorrow — APPROVED** [review]
I'm definitely ok with this going in, but do we want also want to remove all of the stuff that this makes obsolete?

File: packages/protocol/graphics.ts [hunk 1]
```diff
@@ -214,10 +208,6 @@ class VideoPlayer {
       this.commands &&
```
[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/protocol/graphics.ts:208]: I removed all references to the feature flag, and variables that were only used by it.

The `VideoPlayer` class is still referenced a few places (timeline, `components/Video`).

[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/protocol/graphics.ts:208]: Are you suggesting that the `components/Video` component isn't used anymore either? 😮

[the remark this answers is not in the document]
Comment (jcmorrow) [reply packages/protocol/graphics.ts:208]: Maybe it is still doing something, but it seems like at the very least lines 214-221 here cannot be doing anything? Unless the GH diff is tricking my eyes, it looks like they are just assigning to a variable that immediately goes out of scope without doing anything?

[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/protocol/graphics.ts:208]: Oh I see, but the calls it makes to `seek` and `play` are now no longer doing anything.

[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/protocol/graphics.ts:208]: I'll take another pass that removes _more_ of that code!

**Review by jcmorrow — APPROVED** [review]
Now *that's* more like it 🎸

File: packages/protocol/videoNode.ts [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-// without having the two files directly interact with each other.
-let videoNode: HTMLVideoElement | null;
-
-export const setVideoNode = (node: HTMLVideoElement | null) => {
-  videoNode = node;
-};
-
-export const getVideoNode = () => {
-  return videoNode;
-};
```
Comment (bvaughn) [inline packages/protocol/videoNode.ts:14]: Feels particularly nice to get rid of this.

**Pull request merged**